### ISSUE TITLE
timestamp of REV_1991 should be different from REV_1999 and REV_2013

### ISSUE THREAD

**Issue by lkxajd** [dparrini/python-comtrade#22]
timestamp of REV_1991 should be "month/day/year"
before modify:
```
def _read_timestamp(timestamp_line: str, ignore_warnings=False) -> tuple:
    """Process comma separated fields and returns a tuple containing the timestamp
    and a boolean value indicating whether nanoseconds are used.
    Can possibly return the timestamp 00/00/0000 00:00:00.000 for empty strings
    or empty pairs."""
    day, month, year, hour, minute, second, microsecond = (0,)*7
    nanosec = False
    if len(timestamp_line.strip()) > 0:
        values = _read_sep_values(timestamp_line, 2)
        if len(values) >= 2:
            date_str, time_str = values[0:2]
            if len(date_str.strip()) > 0:
                day, month, year = _get_date(date_str)
            if len(time_str.strip()) > 0:
                hour, minute, second, microsecond, \
                    nanosec = _get_time(time_str, ignore_warnings)
```
after modify:
```
def _read_timestamp(timestamp_line: str, rev_year: str, ignore_warnings=False) -> tuple:
    """Process comma separated fields and returns a tuple containing the timestamp
    and a boolean value indicating whether nanoseconds are used.
    Can possibly return the timestamp 00/00/0000 00:00:00.000 for empty strings
    or empty pairs."""
    day, month, year, hour, minute, second, microsecond = (0,)*7
    nanosec = False
    if len(timestamp_line.strip()) > 0:
        values = _read_sep_values(timestamp_line, 2)
        if len(values) >= 2:
            date_str, time_str = values[0:2]
            if len(date_str.strip()) > 0:
                if rev_year in (REV_1999, REV_2013):
                    day, month, year = _get_date(date_str)
                else:
                    month, day, year = _get_date(date_str)
            if len(time_str.strip()) > 0:
                hour, minute, second, microsecond, \
                    nanosec = _get_time(time_str, ignore_warnings)
```

**Comment by engineerjoe440** [comment]
Aha! Good catch! I've worked these suggestions into a [pull request](https://github.com/dparrini/python-comtrade/pull/25) #25 that we can review. I took a little liberty in the way that I created the conditional statement (since 1991 seems to be the only year that the order is different, and everything else is more consistent).

Let's see what @dparrini thinks, and if he is comfortable with the proposed changes, then he'll be able to merge the work!

Thank you for catching this!
